components/events/event.vue

```diff
@@ -3,6 +3,15 @@
     <img :src="img" />
     <h2>{{ title }}</h2>
     <div class="content">
+      <div class="compact" v-if="$store.state.auth.isAuth">
```

Comment:
Bad practice. It's much better to move **isAuth** into _computed_ using mapState